dragonfly/block/break_info.go

```diff
@@ -77,6 +77,14 @@ type BreakInfo struct {
 	// Drops is a function called to get the drops of the block if it is broken using the tool passed. If the
 	// item used to break the block is not a tool, a tool.None is passed.
 	Drops func(t tool.Tool) []item.Stack
+	// XPDrops is the range of XP a block can drop when broken.
+	XPDrops XPDropRange
+}
+
+// XPDropRange is a struct holding the min & max XP drop amounts of blocks.
+type XPDropRange struct {
+	Min,
+	Max int
```

Comment:
I'm thinking it might be a good idea to make this a [2]int, so like
`type XPDropRange [2]int`